intro-HPC/ch_checkpointing.tex

```diff
@@ -1,4 +1,177 @@
+passed between two checkpointed subjobs. However, the \lstinline|user| time should give a good
+indication of the actual time it took to run your command, even if multiple checkpoints were performed.
+
+\subsection{Cleanup checkpoints (\texttt{--cleanup\_after\_restart})}
+
+Specifying this option will make the wrapper script remove the checkpoint files after
+a successful job restart. This may be desirable in cause you're short on storage space.
+
+Note that we don't recommend setting this option, because this way you won't be able
+to resume from the last checkpoint when something goes wrong. It may also prevent
+the wrapper script from reattempting to resubmit a new job in case an infrequent known
+failure occurs. So, don't set this unless you really need to.
+
+\subsection{No cleanup after job completion (\texttt{--no\_cleanup\_chkpt})}
+
+Specifying this option will prevent the wrapper script from cleaning up the
+checkpoints and related information once the job has finished. This may be useful
+for debugging, since this also preserves the \lstinline|stdout|/\lstinline|stderr|
+of the wrapper script.
+
+\subsection{Resuming from last checkpoint (\texttt{--resume})}
+
+The \lstinline|--resume| option allows you to resume a job from the
+last available checkpoint in case something went wrong
+(e.g. accidentally deleting a (sub)job using \lstinline|qdel|, a power outage or other system failure, \ldots).
+
+Specify the job name as returned after submission (and as listed in \texttt{\$VSC\_SCRATCH/chkpt}).
+The full job name consists of the specified job name (or the script name if no job name was specified),
+a timestamp and two random characters at the end, for example \lstinline|my_job_name.20110131_133755.Hk|
+or \lstinline|script.sh.20110131_133755.Hk|.
+
+Note: When resuming from checkpoint, you can change the wall time resources for your job using the \lstinline|--job_time| and
+\lstinline|--chkpt_time| options. This should allow you to continue from the last checkpoint
+in case your job crashed due to an excessively long checkpointing time.
+
+In case resuming fails for you, please contact \hpcinfo, and include the output
+of the \lstinline|csub --resume| command in your message.
+
+\subsection{Specifying what to checkpoint (\texttt{--chkpt\_save\_opt})}
+
+Using \lstinline|--chkpt_save_opt|, you can specify what BLCR should include in
+the checkpoint, i.e. it controls the \lstinline|--save-X| option of \lstinline|cr_checkpoint|.
+The default setting is \lstinline|exe|, which causes the program binary to be included in the
+checkpoint. Specifying \lstinline|none| instead will result in slightly smaller checkpoints,
+however this shouldn't be set if you notice failures to resume from checkpoint
+for example. You can also specify \lstinline|all|, which will significantly increase the size
+of the checkpoints, but may resolve some problems with resuming from checkpoint.
+
+\emph{Don't set this unless you know what you're doing}.
```

Comment:
Actually, just kick out this whole section (see also https://github.com/hpcugent/csub/blob/ede7f1372bf3e9f72037c5716f58c6e2c992c594/csub.py#L730)